features/fixtures/mazerunner/src/main/AndroidManifest.xml

```diff
@@ -11,7 +11,12 @@
 
             </intent-filter>
         </activity>
-        <activity android:name=".SecondActivity"/>
+        <activity android:name=".SecondActivity">
```

Comment:
This now uses an implicit intent rather than an explicit intent. This means scenarios which previously required `SecondActivity` on its classpath can specify `"com.bugsnag.android.mazerunner.UPDATE_CONTEXT"` as the intent instead, and they can be moved to the `mazerunner-scenarios` module instead.